app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(string: str) -> None:
+    words = string.split()
+    old_file_name = words[1]
```

Comment:
Consider using unpacking to assign values from the list to individual variables, as suggested in the checklist.